app/uk/gov/hmrc/customs/declaration/controllers/actionbuilders/batchFileUploadPayloadValidationActions.scala

```diff
@@ -78,7 +78,8 @@ class BatchFileUploadPayloadValidationComposedAction @Inject()(val batchFileUplo
             val files: Seq[BatchFileUploadFile] = (xml \ filesLabel \ "_").theSeq.collect {
               case file =>
                 val fileSequenceNumber = FileSequenceNo((file \ fileSequenceNoLabel).text.trim.toInt)
-                val documentType = DocumentType((file \ documentTypeLabel).text)
+                val documentTypeText = (file \ documentTypeLabel).text
```

Comment:
how about `val maybeDocumentType` which seems to be our convention

Reply:
Done